src/NzbDrone.Core/Parser/Model/SubtitleTitleInfo.cs

```diff
@@ -0,0 +1,34 @@
+using System;
+using System.Collections.Generic;
+using System.Text.RegularExpressions;
+using NzbDrone.Core.Languages;
+
+namespace NzbDrone.Core.Parser.Model
+{
+    public class SubtitleTitleInfo
+    {
+        private static readonly Regex SubtitleTitleRegex = new Regex("((?<title>.+) - )?(?<copy>\\d+)", RegexOptions.Compiled);
+        public List<string> LanguageTags { get; set; }
+        public Language Language { get; set; }
+        public string RawTitle { get; set; }
+        private Lazy<(string Title, int Copy)> TitleAndCopy => new Lazy<(string title, int copy)>(() =>
```

Comment:
The model should be dumb and just a method to store the data parsed elsewhere and passed in. Can this logic instead live in `ParseSubtitleLanguageInformation`?